### DISCUSSION TITLE
Problems updating suricata

### DISCUSSION THREAD

**Discussion by bit1290** [Security-Onion-Solutions/securityonion#6801] [Unsupported Versions] [answered]
Hello,

I'm trying to update suricata but I'm having the following issue:

![image](https://user-images.githubusercontent.com/16745902/148657576-bda1f93e-5941-481a-9e5e-02c45459ec49.png)

Do you know what can be causing this issue?

Thank you! :)

**Comment by TOoSmOotH** [comment]
https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=so-rule-update#so-rule

**Reply by bit1290** [reply]
Hello,

Thank you for your reply, but I'm not trying to make a rule update. I'm trying to update suricata version itself.
Is there something I'm missing?

Thanks.
Regards.

**Reply by jasonish** [reply]
Suricata-Update is a tool to update Suricata rules, not Suricata itself.

**Comment by dougburks** [comment, marked as answer]
Looking at your screenshot, Suricata is showing version 4.1.4. Seems like that box is not running Security Onion 2.
